app/src/test/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharacterPresenterTest.kt

```diff
@@ -33,7 +33,7 @@ class CharacterPresenterTest {
 
         getCharacterServiceUseCase = GetCharacterServiceUseCase(characterServiceImp)
         val subscriptions = mock(CompositeDisposable::class.java)
-        characterPresenter = CharacterPresenter(view, getCharacterServiceUseCase, subscriptions)
+        characterPresenter = CharacterPresenter(view, getCharacterServiceUseCase, subscriptions, CharacterModel("text"), )
```

Comment:
getCharacterServiceUseCase must be a model parameter.. and you can delete the subscriptions here.

Why the model get a hardcoded string? delete it pls

Reply:
Done